litmus-portal/frontend/src/components/Unimodal/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  modalContainer: {
+    width: '70%',
+    padding: '1rem',
+    margin: '2rem auto',
+    background: theme.palette.common.white,
+    borderRadius: 3,
+    textAlign: 'center',
```

Comment:
This is a good change since we have text centred in most of the modals but if you take a look at the User management modal the texts are aligned to the left. You might either want to pass a prop of textAlign to the Unimodal component and set this property here or just completely remove it and expect the developers to wrap the in the styles on the child itself.

>The former one is preferred